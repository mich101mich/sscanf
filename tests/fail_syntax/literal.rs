fn main() {
    let input = "hi";
    let fmt = "{}";
    sscanf::scanf!(input, fmt, std::vec::Vec<usize>);
}
