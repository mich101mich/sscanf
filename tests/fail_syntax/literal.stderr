error: expected string literal
 --> $DIR/literal.rs:4:27
  |
4 |     sscanf::scanf!(input, fmt, std::vec::Vec<usize>);
  |                           ^^^
